docker-compose.yml

```diff
@@ -1,10 +1,10 @@
-version: '3.8'
+version: "3.8"
 services:
   foodloop-postgres:
     image: "postgres:16.1-alpine"
     container_name: foodloop-postgres
     volumes:
-      - ./foodloop-data:/var/lib/postgresql/data
+      - ./foodloop-data:/lib/postgresql/data
```

Comment:
do we push this ver or with `/var/..`

Reply:
okay with either. leaning towards this version since @FooChiHen 's side is not working anyway. This is working for the both of us.

Reply:
go with no var